catroid/src/main/java/org/catrobat/catroid/content/bricks/GlideToBrick.java

```diff
@@ -169,6 +169,10 @@ public void showFormulaEditorToEditFormula(View view) {
 			case R.id.brick_glide_to_edit_text_duration:
 				FormulaEditorFragment.showFragment(view, this, BrickField.DURATION_IN_SECONDS);
 				break;
+
+			default:
+				FormulaEditorFragment.showFragment(view, this, BrickField.DURATION_IN_SECONDS);
+				break;
```

Comment:
When you move the `default` keyword to the corresponding `case` you don't have to implement it twice.
```java
case R.id.brick_glide_to_edit_text_duration:
default:
    // Code
```